### PR TITLE
PIPE2D-1026: fitPfsFluxReference.py: Take `PfsMerged.norm` into account

### PR DESCRIPTION
**Pull request by SogoMineo**
The essential change is lines 362-366. Most of the other changes are required because I use the division operator of the extended PfsMerged class. I take this opportunity and use the division operator also at lines 403-405, though this change might be irrelevant here.

### CODE REVIEW

File: python/pfs/drp/stella/fitPfsFluxReference.py [hunk 1]
```diff
@@ -149,7 +149,7 @@ def run(self, pfsConfig, pfsMerged):
         pfsConfig : `pfs.datamodel.pfsConfig.PfsConfig`
             Configuration of the PFS top-end,
             in which information of broad band fluxes count.
-        pfsMerged : `pfs.datamodel.pfsFiberArraySet.PfsFiberArraySet`
+        pfsMerged : `pfs.drp.stella.PfsFiberArraySet`
```
Comment (PaulPrice) [inline python/pfs/drp/stella/fitPfsFluxReference.py:152]: Isn't it `pfs.drp.stella.datamodel.PfsFiberArraySet`?
I think this change should be a separate commit, as it's not relevant to the normalisation change.

Similarly throughout.

**Review by PaulPrice — APPROVED** [review]

File: python/pfs/drp/stella/fitPfsFluxReference.py (hunk 1 above)
Comment (SogoMineo) [reply python/pfs/drp/stella/fitPfsFluxReference.py:152]: I am splitting the commit into two. In the first commit `pfs.datamodel.PfsFiberArraySet` will be replaced by `pfs.drp.stella.datamodel.PfsFiberArraySet`, and in the second commit the normalisation will be done.
